### ISSUE TITLE
CodeCov/patch/project

### ISSUE THREAD

**Issue by junlee1991** [nus-cs2103-AY2223S2/forum#251]
i'm checking the PR: https://github.com/nus-cs2103-AY2223S2/tp/pull/69 

<img width="674" alt="image" src="https://user-images.githubusercontent.com/77482097/224522066-afc180c7-7c29-477f-99ad-d88ea635c6bb.png">

Any hints how to make the codeCov/project / patch turn into a green tick? 

The logs say Added lines #L93 - L94 were not covered by tests. any way to suppress warning?

**Comment by damithc** [comment]
@junlee1991 The reason is the code coverage is lower than before. You can turn it green by increasing the code coverage. But it is not a requirement from our side to do so (i.e., the coverage level does not affect the grade directly). Also see #246

**Comment by tituswe** [comment]
@damithc Hi prof, in the original version of AB3 that all the teams forked, the UI component was not covered by tests (i.e. the controller classes such as MainWindow.java). I made changes to MainWindow.java and now I start getting warnings by codecov that the methods in that class is not covered by tests. 

Is it possible for us to suppress the warnings where appropriate? Or do we just ignore the codecov?

I have also started a new issue on testing GUI to discuss this question in detail.

**Comment by damithc** [comment]
> Is it possible for us to suppress the warnings where appropriate?

@tituswe Yes, you are free to use CodeCov as you see fit, or not use it at all. Using CodeCov is not a tP requirement.
